packages/preact-query/src/__tests__/useMutation.test.tsx

```
@@ -0,0 +1,1198 @@
+import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest'
+import { fireEvent, render } from '@testing-library/preact'
+import { ErrorBoundary } from './ErrorBoundary'
+import { queryKey, sleep } from '@tanstack/query-test-utils'
```

ESLint

[error] `@tanstack/query-test-utils` import should occur before import of `./ErrorBoundary`

(import/order)